### PR TITLE
Adding readonly prop to avoid BlipSelect click

### PR DESCRIPTION
**Pull request by luccanog**
Added readonly prop to blip-cards to avoid hidding the BlipSelect from QuickReply when visualizing a finished conversation.

### CODE REVIEW

File: package.json
Comment (rodrigoxavier) [inline package.json:3]: Why this is changing from 4 to 11?

File: src/App.vue
Comment (rodrigoxavier) [inline src/App.vue:574]: Shouldn't here be the default value, just like the other properties receive?

File: src/components/BlipSelect.vue
Comment (rodrigoxavier) [inline src/components/BlipSelect.vue:468]: Shouldn't return just checking if is `readonly`? Why is needed to check denying `this.editable`?

File: src/components/BlipSelect.vue
Comment (rodrigoxavier) [inline src/components/BlipSelect.vue:621]: Isn't better `cursor: not-allowed`? This will give a better experience to user displaying a block icon, saying to user that the click is not enabled.

File: src/components/BlipSelect.vue
Comment (luccanog) [reply src/components/BlipSelect.vue:468]: I've considered what @JoaoCMotaJr talked today at the daily. But now with your comment I have some doubts.
If we check the editable, the readonly property will not "override" the behaviour when it is setted true. 
But if we consider that readonly is an optional property and probably only we are going to use it, maybe `&& !this.editable`  it is not necessary

File: src/components/BlipSelect.vue
Comment (luccanog) [reply src/components/BlipSelect.vue:621]: I don't think so, I've considered it, but I think that is not necessary.
For me, the not-allowed seems to mean that the user should perform some action to enable the click, or as the name says, the user is not allowed to click somewhere for some reason. With the default pointer the experience is smoother, since the click is not even a option, the user is not going to be 'not allowed' because the click action doesn't exist.

File: package.json
Comment (luccanog) [reply package.json:3]: I'm trying to match the last release number from github. It seems that the last person forgot to update this package.json version number

File: src/App.vue
Comment (luccanog) [reply src/App.vue]: I've setted false as default value

**Review by rodrigoxavier — APPROVED** [review]

File: src/components/BlipSelect.vue
Comment (mhnakashima) [reply src/components/BlipSelect.vue:621]: It shouldn't be better to show this situation to a PD?

File: src/components/BlipSelect.vue
Comment (JoaoCMotaJr) [reply src/components/BlipSelect.vue]: I accept both possibilities as valid, as long as we put ourselves in the shoes of whoever will try to figure it out this blip cards next. Therefore, both naming seems contradictable so a new user could deduce as well that both flags does not work together

**Review by JoaoCMotaJr — APPROVED** [review]
Please, wait for Building's approval

File: src/App.vue
Comment (luccanog) [reply src/App.vue]: Well noticed. Updated!

**Review by mhnakashima — APPROVED** [review]

**Pull request merged**
